in-market-test/helper-nodes/setup_instructions.md

```diff
@@ -26,9 +26,37 @@ Here are the instructions for setting up node for running IPA.
    
    Link: TBD
 
-3. Docker image already contains public and private keys for encrypting match keys and also, to secure the report files while in transit. Retrieve them
+3. Docker image already contains (default) public and private keys for encrypting match keys and also, 
+   to secure the report files while in transit.
+  
+   1. If you need to install docker, follow the instructions [here](https://docs.docker.com/engine/install/). 
```

Comment:
I still cant follow these instructions. Where do we get the IPA docker image from?

Reply:
It can be build from the source with this command:  `scripts/helper-image.sh --identity 1 --hostname localhost 1`  

Let me add to the instructions.